src/components/Toast/Toast.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState, forwardRef, useImperativeHandle } from 'react';
+import classNames from 'classnames';
+
+export interface ToastProps {
+  id?: string;
+  content: string;
+  duration?: number;
+}
+
+const Toast = ({ id, content, isExiting }: ToastProps & { isExiting: boolean }) => (
+  <div id={id} className={classNames('Layer__toast', { enter: !isExiting, exit: isExiting })}>
+    <p>{content}</p>
+  </div>
+);
+
+export const ToastsContainer = forwardRef((props, ref) => {
```

Comment:
I think `props` is not used, so you can prefix it with underscore `_`
```suggestion
export const ToastsContainer = forwardRef((_props, ref) => {
```